docker/Dockerfile

```diff
@@ -318,11 +318,7 @@ RUN cd /tmp \
 # TestingBot Tunneling #
 # -----------------------#
 # https://testingbot.com/support/other/tunnel
-ENV TB_TUNNEL_URL="https://testingbot.com/tunnel/testingbot-tunnel.jar"
```

Comment:
Probably should leave this in there. You just removed it because we doesn't work at the client site right?

Reply:
Yep. I'll revert this change.

Reply:
I think you could probably add: `ARG testingBotEnabled=true` on the line beforehand. Then you could add `[ "${testingBotEnabled}" == "true" ] && \` before the wget line

Reply:
I've added a pull request to restore this bit with a build arg:
https://github.com/dspasojevic/zalenium/pull/3